test/unit/metrics/test_metricframe_aggregates.py

```diff
@@ -125,6 +125,17 @@ def test_ratio_to_overall(self, metric_fn):
             expected_ratio_overall, rel=1e-10, abs=1e-16
         )
 
+    def test_ratio_zero_divide(self):
```

Comment:
This is for a lurking bug... I can put into a separate PR if there are strong feelings, but it would be simpler to keep here.